### ISSUE TITLE
Q: super resolution regime without up-scaling?

### ISSUE THREAD

**Issue by alexander-rakhlin** [sefibk/KernelGAN#38]
Is KernelGAN applicable for super resolution task without up-scaling, i.e. output = input size ? If yes, how such a regime can be implemented/turned on?

**Comment by sefibk** [comment]
No. You are referring to de-blurring. Not SR. Similar but different
